components/src/status_im/ui/components/react.cljs

```diff
@@ -109,33 +109,32 @@
       (update :style typography/get-style)
       (assoc :max-font-size-multiplier max-font-size-multiplier)))
 
+(defn prepare-nested-text-props [props]
+  (-> props
+      (update :style typography/get-nested-style)
+      (assoc :nested? true)))
+
 ;; Accessor methods for React Components
 (defn text
   "For nested text elements, use nested-text instead"
   ([text-element]
-   [text {} text-element])
+   (text {} text-element))
   ([options text-element]
    [text-class (prepare-text-props options) text-element]))
 
 (defn nested-text
-  ([options & nested-text-elements]
-   (let [options-with-style (prepare-text-props options)]
+  ([{:keys [nested?] :as options} & nested-text-elements]
```

Comment:
so it is `nested-text` and we also can pass `nested?` prop? It is a bit puzzling

Reply:
yeah I added a docstring and moved it out of arg destructuring. It is for internal use only, the reason is that for the child elements you don't want to compute and provide the full styling again and again that is part of what was slowing down rendering so much